src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -116,6 +116,7 @@ var HTMLDOMPropertyConfig = {
     optimum: 0,
     pattern: 0,
     placeholder: 0,
+    playsInline: HAS_BOOLEAN_VALUE,
```

Comment:
Do you know if we need to use the property here? I see that we currently have `muted` requiring the property (`node.muted = true`). Automated testing these things is tricky and I don't have a iOS10 preview to check myself - can you make a small component & run in a seed build that works, testing that toggling between true and false values when rendering actually works as expected?